Common/UnitDefinitions/Radioactivity.json

```diff
@@ -0,0 +1,51 @@
+{
+  "Name": "Radioactivity",
+  "BaseUnit": "Becquerel",
+  "XmlDoc": "Radioactivity is the measure of how radioactive a sample is. The SI unit is the Becquerel (Bq), the non-SI unit is the Curie (Ci)",
+  "BaseDimensions": {
+    "T": -1
+  },
+  "Units": [
+    {
+      "SingularName": "Becquerel",
+      "PluralName": "Becquerels",
+      "BaseUnits": {
+        "T": "Second"
+      },
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Kilo", "Mega", "Giga", "Tera" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Bq" ]
+        },
+        {
+          "Culture": "en-AU",
```

Comment:
`en-US` is the default culture, so you can safely remove `en-AU` here since all the unit abbreviations are identical.